### ISSUE TITLE
Cannot Open JXL After Using ImageMagick For Converting

### ISSUE THREAD

**Issue by cp2732** [ImageMagick/ImageMagick#5785]
### ImageMagick version

7.1.0-52

### Operating system

Windows

### Operating system, version and so on

Windows 10, 22H2

### Description

I tried to use ImageMagick to convert from a TIFF to JXL and, after trying to open the output file using ImageGlass, ImageGlass crashes. I tried to also send the photo to myself using Facebook (which supports JXL previews), and I noticed it looks partially corrupted (it repeats itself three times at the top and the bottom 2/3 portion is very staticky). The photo has dimensions of 1021 x 183 (width x height).

### Steps to Reproduce

Please run the following command in Command Prompt using the input TIFF file:
magick -verbose -define jxl:effort=9 -define jxl:quality=100 "input.tif" "output.jxl"

### Images

I cannot upload either the input TIFF, nor the JXL output, as both file types are not a supported upload file type. I am happy to email the input and output photos for reproducing purposes.
![Screenshot of Output on Facebook Preview](https://user-images.githubusercontent.com/5529206/202927193-01bf287c-ac21-4868-8896-01ac533370f1.PNG)
![Input Tiff File](https://user-images.githubusercontent.com/5529206/202927251-968078ae-bf0f-4a9a-a134-b36b883dd72b.PNG)

**Comment by fmw42** [comment]
What input TIFF file?  Does not seem to be included! Zip it first, then include it. I do not think this forum supports TIFF upload directly. Alternately, post it to some free hosting service that supports TIFF and put the URL here

**Comment by cp2732** [comment]
@fmw42 I have zipped both the input and output files. Please see them attached to this comment.
[input.zip](https://github.com/ImageMagick/ImageMagick/files/10050795/input.zip)
[output.zip](https://github.com/ImageMagick/ImageMagick/files/10050796/output.zip)

**Comment by fmw42** [comment]
This command works fine for me on IM 7.1.0-52 Q16-HDRI on Mac OSX Monetary

`magick -verbose input.tif -define jxl:effort=9 -define jxl:quality=100 output.jxl`

The output opens fine for me in GraphicConverter 11 on the Mac

Here is my output. See if it opens for you properly on your other software.

[output.jxl.zip](https://github.com/ImageMagick/ImageMagick/files/10050809/output.jxl.zip)


Your output also opens fine for me in GraphicConverter.   Perhaps the issue is with your other tool for viewing it.

**Comment by cp2732** [comment]
I tried to convert it separately on a Mac (same OS and IM version as you) and I got the same exact output file I got from running the same command I had on Windows. I tried to open it with JXLook on my Mac and, as you are seeing with GraphicConverter 11, it opens fine. Both output file sizes are 25,101 bytes and, after using a file diff tool, they also have the same contents. I guess both ImageGlass and Facebook have problems opening certain JXL files still. I will reach out to both separately and report the issues I'm seeing with both.
<img width="1440" alt="Screen Shot 2022-11-20 at 7 14 46 PM" src="https://user-images.githubusercontent.com/5529206/202935298-e1e65807-4297-4b37-a508-37f764c16f2d.png">

Thanks for the help in identifying it was an issue with the image applications themselves and not ImageMagick in this case.
